perllib/FixMyStreet/Cobrand/Thamesmead.pm

```diff
@@ -77,6 +77,10 @@ sub example_places {
     return [ 'Glendale Way', 'Manorway Green' ];
 }
 
+sub get_body_handler_for_problem {
```

Comment:
Please log a ticket about how things will need to be changed to have this work if reports can be made on .com so we don't forget.
I think (though not totally sure) the only issue here is if someone looks at a Peabody report on .com, it'll use either Greenwich/Bexley's cobrand for `is_council_with_case_management` (neither of which use that). You could add an exception similar to TfL's to the UK.pm get_body_handler_for_problem, which I guess would fix that for now.